Ships as Raw.txt

```diff
@@ -654,151 +530,247 @@ FrontArcConjunction|65
 RearArcConjunction|49
 //How far from the rear of the ship do the rear hull zone lines meet//
 
-
-Name|Imperial II Class Star Destroyer 
+Name|Munificent-class Star Frigate
+Keyword| Droid Bombard Transport
+Faction|Separatist
 Keyword|None
-Faction|Imperial
-Size|Large
-Points|120
-Hull|11
-Defense token| Rd Br Br Cn 
-Command|3
-Squad|4
+Size|Medium
+Points|73
+Hull|6
+Defense token| Sal Rd Br
+Command|2
+Squad|3
 Eng|4
-Nav chart 
+Nav chart
 Speed 1|1
-Speed 2|1 1
-Speed 3|0 1 1
+Speed 2|1 2
+Speed 3|-
 Speed 4|-
-FrontArc 4 |RRRRBBBB
-SideArc 3 |RRBB
-RearArc 2 |RBB
-AntiSquadron|BB
-Upgrade|Title Officers Defensive Offensive Weapons Team Turbo Laser Defensive Ion Cannon
-FrontArcOffset|41
+FrontShields|4 FrontArmament|RRBB
+SideShields|3 SideArmament|RRR
+RearShields|2 RearArmament|RR
+AntiSquadron|R
+Upgrade|Title Officers TurboLaser Defensive Suport Team Ion Cannon
+FrontArcOffset| 7.9
 //How far from the front corner does the hull zone line hit the cardboard edge//
-RearArcOffset|15
+RearArcOffset|3.2
 //How far from the rear corner does the hull zone line hit the cardboard edge//
-FrontArcConjunction|79
+FrontArcConjunction|53
 //How far from the front of the ship do the front hull zone lines meet//
 RearArcConjunction|49
 //How far from the rear of the ship do the rear hull zone lines meet//
 
-Name|Imperial I Class Star Destroyer 
-Keyword|None
-Faction|Imperial
-Size|Large
-Points|110
-Hull|11
-Defense token| Rd Br Br Cn 
-Command|3
-Squad|4
-Eng|4
-Nav chart 
-Speed 1|1
+
+Name|Hardcell-class Transport
+Keyword|Droid Transport
+Faction|Separatist
+Size|Small
+Points|47
+Hull|5
+Defense token|Br Rd Ev
+Command|1
+Squad|2
+Eng|3
+Nav chart
+Speed 1|2
 Speed 2|1 1
-Speed 3|0 1 1
+Speed 3|1 1 0
 Speed 4|-
-FrontArc 4 |RRRBBKKK
-SideArc 3 |RRKK
-RearArc 2 |RBB
-AntiSquadron|BK
-Upgrade|Title Officers Defensive Offensive Offensive Weapons Team Turbo Laser Defensive Ion Cannon
-FrontArcOffset|41
+FrontShields|3 FrontArmament|RRB
+SideShields|1 SideArmament|B
+RearShields|2 RearArmament|-
+AntiSquadron|B
+Upgrade|Title Support Fleet Support Officer Officer
+FrontArcOffset| 13.1
 //How far from the front corner does the hull zone line hit the cardboard edge//
-RearArcOffset|15
+RearArcOffset| 13.1
 //How far from the rear corner does the hull zone line hit the cardboard edge//
-FrontArcConjunction|79
+FrontArcConjunction|32
 //How far from the front of the ship do the front hull zone lines meet//
-RearArcConjunction|49
+RearArcConjunction|31
 //How far from the rear of the ship do the rear hull zone lines meet//
+//This ship has a 7mm section between the rear and front conjuctions.//
 
+Name|Hardcell-class Battle Refit
+Keyword|Droid Transport
+Faction|Separatist
+Size|Small
+Points|52
+Hull|5
+Defense token|Br Rd Ev
+Command|1
+Squad|2
+Eng|3
+Nav chart
+Speed 1|2
+Speed 2|1 1
+Speed 3|1 1 0
+Speed 4|-
+FrontShields|3 FrontArmament|RRR
+SideShields|1 SideArmament|R
+RearShields|2 RearArmament|-

```

Comment:
These 3 lines should be 6.

ie 
FrontShields|3
FrontArmament|RRR

Since we can't have more than 1 "pipe" (`|`) per line

Pretty sure I saw this in a few other spots